src/server.c

```diff
@@ -58,29 +58,24 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
     ///////////////////
 
     // Build HTTP response and store it in response
-    // body = "<div>Hello</div>";
-    int body_length = strlen(body);
-
-    printf("body:%s\n%i",body, body_length);
 
+    // gets current time as a string
     time_t rawtime;
     struct tm * timeinfo;
-
     time (&rawtime);
     timeinfo = localtime (&rawtime);
+
     char * current_date = asctime(timeinfo);
-    printf ("Current local time and date: %s", current_date);
 
-    int response_length = sprintf(&response, "HTTP/1.1 200 OK\n"
+    int response_length = sprintf(&response, "%s\n"
             "Date: %s"
-            "Connection: keep-alive\n"
+            "Connection: close\n"
             "Content-Length: %d\n"
             "Content-Type: text/html\n"
             "\n"
-            "%s", current_date, body_length,  body);
-
-    printf("sr sprintf:%s\n-----------\n", response);
+            "%s", header, current_date, content_length,  body);
```

Comment:
I mentioned this today but I think having it in writing will help reference later: this will handle text files fine, but it won't work for jpgs or pngs. You'll have to figure out another way to copy the body into the response if you want to do that.